internal/source/conf.go

```diff
@@ -71,18 +71,18 @@ func (s S) String() string {
 	return SourceToString[s]
 }
 
-func readConf(path, alias string, c chan string) error {
+func readConf(path, alias string, c chan string) {
 	f, err := os.Open(path)
 	if err != nil {
 		c <- ""
-		return nil
+		return

```

Comment:
I think that when there is an error in reading the configuration file, it should be to throw an error, the user is using the incorrect configuration, he can know exactly what caused the error.

e.g. 
 viper would return an error when reading the configuration file failed.
https://github.com/kubesphere/kubekey/blob/07682c08d72b29021e57f091a9e10bd2310aa24a/cmd/kk/pkg/config/from_cluster.go#L178

Reply:
I have caught the error